src/pages/IndexPage.tsx

```diff
@@ -0,0 +1,36 @@
+export function IndexPage() {
+  return (
+    <div className="h-screen flex flex-col">
```

Comment:
prefer `h-full` in case this page ever exists within a parent. `h-full` will set height to 100%, which in the right kind of container will take up 100% of available space, while h-screen will always try to take 100vh.